source/clear-linux/guides/maintenance/download-verify-decompress.rst

```diff
@@ -0,0 +1,180 @@
+.. _download-verify-decompress-linux:
+.. _download-verify-decompress-mac:
+.. _download-verify-decompress-windows:
+
+
+Download, verify, and decompress a |CL-ATTR| image
+##################################################
+
+This guide describes the types of |CL| images available, where to download
+them, how to verify the integrity of an image, and how to decompress it.
+
+
+Image types
+***********
+
+.. include:: ../../reference/image-types.rst
+   :start-after: image-types-content:
+
+Instructions for each operating system
```

Comment:
Could we move the "follow instructions for your OS" bit to the intro and then do away with this heading entirely? Then update each OS set of instructions to use the same heading level as Image types (eg use ***** to underline). Then the page outline from the local TOC will read: 
-Image types
-Linux instructions
-macOS instructions
-Windows instructions

And users can jump to their section from the top if desired.